op-challenger/config/config.go

```diff
@@ -258,7 +258,7 @@ func (c Config) Check() error {
 			return err
 		}
 	}
-	if c.TraceTypeEnabled(types.TraceTypeCannon) || c.TraceTypeEnabled(types.TraceTypePermissioned) {
+	if c.TraceTypeEnabled(types.TraceTypeCannon) || c.TraceTypeEnabled(types.TraceTypePermissioned) || c.TraceTypeEnabled(types.TraceTypeCannonKona) {
```

Comment:
I think this is wrong - we should have a `CannonKona vm.Config` field added instead of reusing the `Cannon` field - like we have `Asterisc` and `AsteriscKona`. The vm.Config contains the server to run which needs to be op-program for Cannon and Kona-server for CannonKona.

Reply:
Done! https://github.com/ethereum-optimism/optimism/pull/17249/commits/0b509db2cb58dc5c69f663453a0bbb07ab970c33